src/main/kotlin/uk/gov/justice/digital/hmpps/manageoffencesapi/entity/Offence.kt

```diff
@@ -20,11 +20,18 @@ data class Offence(
   val revisionId: Int? = null,
   val startDate: LocalDate? = null,
   val endDate: LocalDate? = null,
-  val homeOfficeStatsCode: String? = null,
+  val category: Int? = null,
+  val subCategory: Int? = null,
+  val actsAndSections: String? = null,
   val changedDate: LocalDateTime? = null,
   val createdDate: LocalDateTime = LocalDateTime.now(),
   val lastUpdatedDate: LocalDateTime = LocalDateTime.now(),
 ) {
   val statuteCode
     get() = code.substring(0, 4)
+  val homeOfficeStatsCode: String?
+    get() {
+      if (category == null && subCategory == null) return null
+      return category?.toString().orEmpty().padStart(3, '0') + "/" + subCategory.toString().padStart(2, '0')
```

Comment:
Can you have null category with non-null subCategory? There's some SQL down the bottom of this PR that looks like that...

Reply:
yes you can.. (although that shouldn't happen, but is possible because it's not validated by SDRS ).
Ive just had a conversation with Joel about this and we will report the bad data to SDRS so they can correct it . However we still cater for it. 

Will add some specific unit tests for this to test all  permutations